src/validation/resolvers.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs')
```

Comment:
I'm going to postpone reviewing this file until we have the spec as an RFC in https://github.com/graphprotocol/rfcs/.